internal/http/services/owncloud/ocdav/net/net.go

```diff
@@ -110,6 +118,41 @@ func ParseDepth(s string) (Depth, error) {
 	case DepthInfinity.String():
 		return DepthInfinity, nil
 	default:
-		return "", fmt.Errorf("invalid depth: %s", s)
+		return "", errors.Wrapf(ErrInvalidHeaderValue, "invalid depth: %s", s)
+	}
+}
+
+// ParseOverwrite parses the overwrite header value defined in https://datatracker.ietf.org/doc/html/rfc4918#section-10.6
+// Valid values are "T" and "F". An empty string will be parse to true.
```

Comment:
Not quite true. `"f"` and `"t"` are also valid. 

I wonder if we should use `strconv.ParseBool` (then `true` would also be valid)?

Reply:
> I wonder if we should use strconv.ParseBool (then true would also be valid)?
No, I would stick to the defined values from the RFC, I'll change the code to not include the upper casing.